test/videojs-hls_test.js

```diff
@@ -2254,6 +2260,8 @@ test('calling play() at the end of a video resets the media index', function() {
 
   player.play();
   strictEqual(player.hls.mediaIndex, 0, 'index is 1 after the first segment');
+
+  player.el().querySelector('.vjs-tech').vjs_getProperty = function() {};
```

Comment:
This can be rewritten a bit more succinctly as `player.tech.el().vjs_getProperty`